### PR TITLE
grey_replaceText() add UITextViewTextDidChangeNotification

### PR DESCRIPTION
**Pull request by AzraelWang**
why grey_replaceText() send UITextFieldTextDidChangeNotification,but doesn't send UITextViewTextDidChangeNotification, when UITextView replaceText, test doesn't work.
Please check.

### CODE REVIEW

File: EarlGrey.xcodeproj/project.xcworkspace/contents.xcworkspacedata [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (khandpur) [inline EarlGrey.xcodeproj/project.xcworkspace/contents.xcworkspacedata]: Please revert this entire file. It's not required.

File: EarlGrey/Action/GREYActions.m [hunk 2]
```diff
@@ -401,6 +402,13 @@ + (void)grey_setText:(NSString *)text onWebElement:(id)element {
                                           object:element];
         [NSNotificationCenter.defaultCenter postNotification:notification];
       }
+        
+      if (elementIsUITextView) {
+        NSNotification *notification =
+        [NSNotification notificationWithName:UITextViewTextDidBeginEditingNotification
```
Comment (khandpur) [inline EarlGrey/Action/GREYActions.m]: please format this by indenting it 4 more spaces

File: EarlGrey/Action/GREYActions.m [hunk 3]
```diff
@@ -415,7 +423,13 @@ + (void)grey_setText:(NSString *)text onWebElement:(id)element {
                                           object:element];
         [NSNotificationCenter.defaultCenter postNotification:notification];
       }
-
+      if (elementIsUITextView) {
+        NSNotification *notification =
+        [NSNotification notificationWithName:UITextViewTextDidChangeNotification
```
Comment (khandpur) [inline EarlGrey/Action/GREYActions.m]: please format this by indenting it 4 more spaces

File: EarlGrey/Action/GREYActions.m [hunk 4]
```diff
@@ -427,6 +441,13 @@ + (void)grey_setText:(NSString *)text onWebElement:(id)element {
                                           object:element];
         [NSNotificationCenter.defaultCenter postNotification:notification];
       }
+      if (elementIsUITextField) {
+        NSNotification *notification =
+        [NSNotification notificationWithName:UITextViewTextDidEndEditingNotification
```
Comment (khandpur) [inline EarlGrey/Action/GREYActions.m]: please format this by indenting it 4 more spaces

**Comment by khandpur** [comment]
It looks like you also need to sign the CLA before we can merge this code.

File: EarlGrey.xcodeproj/project.xcworkspace/contents.xcworkspacedata (hunk 1 above)
Comment (AzraelWang) [reply EarlGrey.xcodeproj/project.xcworkspace/contents.xcworkspacedata]: OK. I will sign the CLA first.

**Comment by AzraelWang** [comment]
I signed it!

**Comment by xiaozhuTeam** [comment]
I signed it!

**Comment by AzraelWang** [comment]
I signed it!

**Comment by xiaozhuTeam** [comment]
Yes. I am OK with that. @googlebot

**Comment by AzraelWang** [comment]
#642

**Comment by AzraelWang** [comment]
@khandpur should I push request with another account？It seem that I commit my code with my 'xiaozhuTeam' account, but I push request with this one.

**Comment by khandpur** [comment]
@xiaozhuTeam Thank you for the consent. The issue arose because the email associated with the commit is different from what you put in the CLA. It's fine for now because you explicitly gave us your consent but in future please make sure the same email is used :)

File: EarlGrey/Action/GREYActions.m [hunk 5]
```diff
@@ -389,18 +389,24 @@ + (void)grey_setText:(NSString *)text onWebElement:(id)element {
...
       if (elementIsUITextField) {
         NSNotification *notification =
             [NSNotification notificationWithName:UITextFieldTextDidBeginEditingNotification
                                           object:element];
         [NSNotificationCenter.defaultCenter postNotification:notification];
       }
+      if (elementIsUITextView) {
+        NSNotification *notification =
+            [NSNotification notificationWithName:UITextViewTextDidBeginEditingNotification
+                                      object:element];
```
Comment (khandpur) [inline EarlGrey/Action/GREYActions.m]: please also align color (:) with the one above.

File: EarlGrey/Action/GREYActions.m [hunk 6]
```diff
@@ -415,7 +421,13 @@ + (void)grey_setText:(NSString *)text onWebElement:(id)element {
                                           object:element];
         [NSNotificationCenter.defaultCenter postNotification:notification];
       }
-
+      if (elementIsUITextView) {
+        NSNotification *notification =
+            [NSNotification notificationWithName:UITextViewTextDidChangeNotification
+                                      object:element];
```
Comment (khandpur) [inline EarlGrey/Action/GREYActions.m]: please also align color (:) with the one above.

File: EarlGrey/Action/GREYActions.m (hunk 5 above)
Comment (AzraelWang) [reply EarlGrey/Action/GREYActions.m]: @khandpur  done

**Comment by khandpur** [comment]
uh oh...looks like there were a lot more commits that went in and now it has conflicts.

**Pull request closed without merging**